ansible/inventory

```diff
@@ -0,0 +1,9 @@
+# Ansible inventory for using p3-appliances in the ALaSKA production
+# environment.
+
+# Ansible Shade uses OpenStack clients running locally
+[openstack]
+localhost ansible_connection=local ansible_python_interpreter=python
+
+[alaska:children]
```

Comment:
No need for Alaska here.  After the PR merges, this can be removed in a subsequent PR.